var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -51,15 +51,15 @@ class Mfem(Package):
     # If this quick verification procedure fails, additional discussion
     # will be required to verify the new version.
 
-    version('3.3.2', 
+    version('3.3.2',
             '01a762a5d0a2bc59ce4e2f59009045a4',
             url='https://goo.gl/Kd7Jk8', extension='.tar.gz',
```

Comment:
Why are short URLs used? It looks like `http://mfem.github.io/releases/mfem-3.3.2.tgz` works just as well.

Reply:
https://github.com/spack/spack/pull/2972#issuecomment-279227646

Reply:
Also see comment directly above this in the `mfem` package.

Reply:
Maybe a comment in the code mentioning why this is here would be good?